src/SimpleCalculator.Web/Views/Shared/_CalculationResults.cshtml

```diff
@@ -0,0 +1,52 @@
+@model List<SimpleCalculator.DataAccess.Model.CalculationResultEntity>
+
+<h3>Calculation Results</h3>
+
+<hr />
+
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                Id
+            </th>
+            <th>
+                Math Operator
+            </th>
+            <th>
+                First Number
+            </th>
+            <th>
+                Second Number
+            </th>
+            <th>
+                Result
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        @if (Model != null)
+        {
+            @foreach (var results in Model)
```

Comment:
`results` variable has incorrect name, because it's just one item. Also, it would be good to name to understand what it is